### PR TITLE
added core/key values, quantum, netlix/airbnb/uber for, cognitive/ion, focus, wisdom

### PR DESCRIPTION
**Pull request by crawftv**
added core/key values, quantum, netlix/airbnb/uber for, cognitive/ion, focus, wisdom

### CODE REVIEW

File: src/terms.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ export default [
     '(market|goal|community|quality|results|sales|user|customer' +
         '|subject|role|service|client|process|business)' +
         '.(centric(ity)?|facing|oriented|driven|focused|assessment|service|process)',
+    '(core|key)' + 'values',
```
Comment (mourner) [inline src/terms.js]:
```suggestion
    '(core|key) values',
```

File: src/terms.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ export default [
...
+    '(uber|netflix|lyft|airbnb)'+ 'for',
```
Comment (mourner) [inline src/terms.js]:
```suggestion
    '(uber|netflix|lyft|airbnb) for',
```

File: src/terms.js [hunk 3]
```diff
@@ -119,6 +122,7 @@ export default [
     'fault.tolerant',
     'first.to.market',
     'flexibility',
+    'focus'
```
Comment (mourner) [inline src/terms.js]:
```suggestion
    'focus',
```

File: src/terms.js [hunk 4]
```diff
@@ -237,6 +241,7 @@ export default [
     'push the envelope',
     'push.back',
     'productivity',
+    'qauntum',
```
Comment (mourner) [inline src/terms.js]:
```suggestion
    'quantum',
```

**Comment by brianmcfadden** [comment]
'values?' looks like it's already in the bullshit terms list.  I think i would be good to remove 'values?' in favor of this '(core|key).values' term.  The term "values" has other contexts (mathematical, programming) where it's not quite so bullshitty.  The '(core|key)' makes it more specific, which is good.

**Pull request closed without merging**